Repository: plone.app.dexterity


Branch: refs/heads/master
Date: 2016-06-17T16:41:39+02:00
Author: Fred van Dijk (fredvd) <fredvd@gmail.com>
Commit: https://github.com/plone/plone.app.dexterity/commit/136ecc5c7d83646075aa1ccb9053d8dba5ec3277

fix typo

And try to get the message the same as  plone.schemaeditor/plone/schemaeditor/interfaces.py:138

Files changed:
M plone/app/dexterity/interfaces.py

diff --git a/plone/app/dexterity/interfaces.py b/plone/app/dexterity/interfaces.py
index f08743c..31912e6 100644
--- a/plone/app/dexterity/interfaces.py
+++ b/plone/app/dexterity/interfaces.py
@@ -27,7 +27,7 @@ class ITypeSchemaContext(Interface):
 
 class InvalidIdError(schema.ValidationError):
     __doc__ = _(
-        u'Please use only letters, numbers, and the following characters: .-_')
+        u'Please use only letters, numbers and the following characters: .-_')
 
 # a letter followed by letters, numbers, period, hyphen, or underscore
 ID_RE = re.compile(r'^[a-z][\w\d\.-]*$')


